.github/workflows/prepare_release.yml

```diff
@@ -0,0 +1,53 @@
+name: Prepare release
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  prepare_release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set RELEASE_VERSION
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Prepare release
+
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v2
+        with: python-version:3.8
```

Comment:
This needs to become
```
        with: 
          python-version: 3.8
```
Notice the space also between `python-version:` and `3.8`; it's needed.

Reply:
This is fixed now.

Reply:
Fixed.